backend/tests/v5/e2e/routes/users.test.js

```diff
@@ -433,6 +441,82 @@ const testDeleteApiKey = () => {
 	});
 };
 
+const testForgotPassword = () => {
+	describe('Send forgot password email', () => {
+		test('should fail if a username or email is not provided', async () => {
+			const res = await agent.post('/v5/user/password').expect(templates.invalidArguments.status);
+			expect(res.body.code).toEqual(templates.invalidArguments.code);
+		});
+
+		test('should return ok even if user does not exist', async () => {
+			await agent.post('/v5/user/password').send({ user: 'non existing user' })
+				.expect(templates.ok.status);
+		});
+
+		test('should send forgot password email with valid username', async () => {
+			await agent.post('/v5/user/password').send({ user: testUser.user })
+				.expect(templates.ok.status);
+		});
+	});
+};
+
+const testResetPassword = () => {
+	describe('Reset user password', () => {
+		test('should fail if a token is not provided', async () => {
+			const res = await agent.put('/v5/user/password').send({ newPassword: 'Abcdefghijk1!', user: testUserWithToken.user })
+				.expect(templates.invalidArguments.status);
+			expect(res.body.code).toEqual(templates.invalidArguments.code);
+		});
+
+		test('should fail if a new password is not provided', async () => {
+			const res = await agent.put('/v5/user/password').send({ token: 'some random token', user: testUserWithToken.user })
+				.expect(templates.invalidArguments.status);
+			expect(res.body.code).toEqual(templates.invalidArguments.code);
+		});
+
+		test('should fail if user is not provided', async () => {
+			const res = await agent.put('/v5/user/password').send({ newPassword: 'Abcdefghijk1!', token: 'some random token' })
+				.expect(templates.invalidArguments.status);
+			expect(res.body.code).toEqual(templates.invalidArguments.code);
+		});
+
+		test('should fail if the new password is too weak', async () => {
+			const res = await agent.put('/v5/user/password').send({ newPassword: 'Abcdef1!', token: 'some random token', user: testUserWithToken.user })
+				.expect(templates.invalidArguments.status);
+			expect(res.body.code).toEqual(templates.invalidArguments.code);
+		});
+
+		test('should fail if user is not found', async () => {
+			const res = await agent.put('/v5/user/password').send({ newPassword: 'Abcdefghijk1!', token: 'some random token', user: 'invalid user' })
```

Comment:
return invalid token here

we don't want people to use this endpoint to figure out usernames